source/clear-linux/get-started/bare-metal-install-beta-intro/bare-metal-install-beta-intro.rst

```diff
@@ -0,0 +1,84 @@
+.. _bare-metal-install-beta-intro: 
+
+Install from the live desktop image beta
```

Comment:
Install from the live desktop beta image

(sorry to give more changes here, Michael!)